### PR TITLE
Chore/#tc 37 create pull request template

### PR DESCRIPTION
**Pull request by CristiPV**
### This pull request closes:

✔️ Closes: #207 
🏁 Jira Issue: [TC-37](https://treecreate.atlassian.net/jira/software/c/projects/TC/boards/1?modal=detail&selectedIssue=TC-37)

### Which service does this issue affect:

- [x] 🛰️ API 
- [x] 🛒 webstore
- [x] 🎓 admin-page

### Types of changes

- [ ] 🐛 Bug fix (a non-breaking change which fixes an issue)
- [x] 🧱 New feature (a non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)

### Documentation Updates

- [x] 📂 No need for updates
- [ ] 📁 Requires an update ( _not done within this issue_ )
- [ ] 🗃️ Has been updated ( _done within this issue_ )

### Testing checklist

- [x] 💪 Manually tests
- [ ] 🔧 Automatic tests

#### Browser compatibility - _**[**Frontend only**]**_

- [ ] Tested on Safari
- [ ] Tested on Chrome
- [ ] Tested on Mozilla Firefox

### What changes have been done within this issue?

- I have updated the `.gitignore` to include the *vscode* workspace file, and added *vscode* configurations.
- I have created a pull request template markdown file.

### How should this be manually tested?

By either creating a pull request or by going into the `./.github/pull_request_template.md` and checking out the file.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+### Which service(s) does this issue affect:
+
+- [ ] 🛰️ API
+- [ ] 🛒 webstore
+- [ ] 🎓 admin-page
+
+### Types of changes
+
+- [ ] 🐛 Bug fix (a non-breaking change which fixes an issue)
```
Comment (Kwandes) [inline .github/pull_request_template.md:14]: We have other types of changes like ci, refactor etc
This pr is not adding a new feature, it is either a chore or docs

I would add an extra option called `other`

**Review by Kwandes — CHANGES_REQUESTED** [review]
Looks good, there are only 2 minor changes left

- Add extra option to types of changes
- Remove the change to package-lock.json (need to revert that commit and resubmit the change to gitignore).
  > In the future avoid including changes to package.json etc in commits for other changes so actions like this are easier to do

File: .github/pull_request_template.md (hunk 1 above)
Comment (CristiPV) [reply .github/pull_request_template.md:14]: It is literally a new feature, but I get what you mean.

**Comment by Kwandes** [comment]
The jira issue id in the pull request title should have a dash (title should be `Chore/#tc-37 create pull request template`)
This is defined in the development pipeline. It also ensures that the pull request gets correctly linked in Jira, although that is also redundantly set in the closes section

**Review by Teodor25 — APPROVED** [review]

**Review by Kwandes — APPROVED** [review]
🚀

**Pull request merged**